File: src/main/java/task_4/Square.java
package task_4;

public class Square implements ICalculateShape {
    int sideLength;

    public void setSideLength(int sideLength) {
        this.sideLength = sideLength;
    }

    @Override
    public int area() {
        return this.sideLength * this.sideLength;
    }
}

File: src/main/Python/task_4/square.py
from shape import Shape


class Square(Shape):
    def __init__(self, side_length):
        self.side_length = side_length

    def set_side_length(self, side_length):
        self.side_length = side_length

    def area(self, side_length):
        return side_length * side_length
